lib/pages/surat_page_v3/surat_page_state_notifier.dart

```diff
@@ -716,6 +709,10 @@ class SuratPageStateNotifier extends BaseStateNotifier<SuratPageState> {
     return false;
   }
 
+  void refresh() {
+    state = state.refresh();
+  }
+
   void _setIsBookmarkChanged() {
     _isBookmarkChanged = true;
   }
```

Comment:
oot, with a single file reaching 718 lines, I think moving forward, we should allocate time to refactor files that exceed 500 lines(?) and split them into several files.

Reply:
updated @indraseptama

Reply:
setuju kalo yang state notifier ini agak susah dipecah, tapi mungkin dari sisi widget ada potensi untuk dipecah lalu kita gunain state yang relevan per masing masing widget. Perlu dipikirin strategi nya sih, tapi mungkin ini bisa jadi sedikit acuan https://riverpod.dev/docs/providers/provider, yang bagian bawah ada section buat improve lagi code nya. Dipecah gunain spesifik provider yang relevan dengan masing masing widget buat ngurangin rebuild di widget tersebut. Tapi mungkin ini bisa dibahas di next meeting FE.